tests/hash-graph-http/tests/ambiguous.http

```diff
@@ -130,6 +133,9 @@ X-Authenticated-User-Actor-Id: {{account_id}}
     "$schema": "https://blockprotocol.org/types/modules/graph/0.3/schema/data-type",
     "kind": "dataType",
     "$id": "http://localhost:3000/@alice/types/data-type/miles/v/1",
+    "allOf": [{
+        "$ref": "https://blockprotocol.org/@blockprotocol/types/data-type/number/v/1"
```

Comment:
Conceptually this should have `length` as a parent like the others above... if that is going to mess up the tests then maybe we can pick some other example than `miles` so that some future person doesn't wonder why it has `number` and not `length` (e.g. `integer`)

Reply:
I agree. I mean we should move those tests to Rust anyway and then we'd use the test data as defined there, however, until we have it it's better to have it consistent. [`2e7e6e2` (#5240)](https://github.com/hashintel/hash/pull/5240/commits/2e7e6e204ea746842c641048e51122c88496e419)